packages/perseus/src/widgets/interactive-graphs/backgrounds/grid.test.ts

```diff
@@ -1,72 +0,0 @@
-import {lineLabelText} from "./grid";
```

Comment:
This is testing a function that was not being used, and has been deleted as part of this PR.